src/App.tsx

```diff
@@ -104,7 +135,7 @@ function App() {
     }
   }, [connection, name, hue]);
 
-  function handleChangeLanguage(language: string) {
+  function handleChangeLanguage(language: Language) {
```

Comment:
A `useEffect` with a dependency on language would probably be fine, but it would require an extra render. Also need to just make sure that we're not showing extraneous messages on first launch of the page.